src/ad-services/bill-the-lizard/index.ts

```diff
@@ -400,19 +400,13 @@ export class BillTheLizard {
 	 * @returns {number | undefined}
 	 */
 	getPreviousPrediction(startId, callIdBuilder, modelName) {
```

Comment:
What about
```suggestion
	getPreviousAndReusablePrediction(modelName) {
```
First two arguments are not needed. And This method name seems to be more descriptive. It will require bumping "Major" version because of changed API.